src/app/tests/integration/entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+service dbus start
+sleep 1
```

Comment:
could we remove this sleep (ideally there should be no sleeps, but not sure if possible)? unsure if avahi-daemon and otbr depend on dbus.

Reply:
just remove that sleep! Done, thanks